features/_secureboot/file.exclude

```diff
@@ -1 +1 @@
-/boot/efi/*
+/boot/efi/syslinux/*
```

Comment:
why limit this to syslinux files only? the idea behind the secure boot feature is to *only* support secure boot (to reduce image size by avoiding duplicated data for kernel and initrd as both standalone and inside the combined efi image) &rArr; only the combined efi image should be needed on the efi system partion, which gets build later on (`makepart` / `makesecureboot`), so all other files can safely be cleared from the efi partition entirely

Reply:
I reverted this change back to `/boot/efi/*` but I had to adjust `makepart` since it tried to touch two files in the `/boot/efi/loader` directory which didn't exists because of the `file.exclude`.
* https://github.com/gardenlinux/gardenlinux/blob/feature/1107_add_secureboot_feature/bin/makepart#L28-L30